### PR TITLE
add db image version to details

### PR DESCRIPTION
**Pull request by grrowe**
added image version to database detail/list per: https://www.notion.so/aptible/App-UI-Bugs-Ideas-a386eb0d325d410a93717c6d91de0466?pvs=4#37a73d152ebc4fa0a69984f18ce7a297

### CODE REVIEW

File: src/ui/layouts/database-detail-layout.tsx [hunk 1]
```diff
@@ -75,13 +81,11 @@ export function DatabaseHeader({
         <DetailInfoItem title="Memory Limit">
           {metrics.totalMemoryLimit / 1024} GB
         </DetailInfoItem>
-
         <DetailInfoItem title="Type">
-          {capitalize(database.type)}
+          {`${capitalize(database.type)} ${image.version || ""}`}
```
Comment (neurosnap) [inline src/ui/layouts/database-detail-layout.tsx:85]: Do you think we should have a helper function inside `src/deploy/images` to generate this text for us?

**Review by neurosnap — APPROVED** [review]
one question, non blocking

**Review by joeclayallday — APPROVED** [review]

File: src/ui/layouts/database-detail-layout.tsx (hunk 1 above)
Comment (grrowe) [reply src/ui/layouts/database-detail-layout.tsx:85]: Probably wouldn't be a bad idea, especially if we start wanting to put version in future places. I will create a helper for this

**Review by joeclayallday — APPROVED** [review]

File: src/database/index.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { capitalize } from "@app/string-utils";
```
Comment (neurosnap) [inline src/database/index.ts:1]: can this go to `src/deploy/database`?

File: src/ui/layouts/database-detail-layout.tsx [hunk 3]
```diff
@@ -75,13 +81,12 @@ export function DatabaseHeader({
         <DetailInfoItem title="Memory Limit">
           {metrics.totalMemoryLimit / 1024} GB
         </DetailInfoItem>
-
         <DetailInfoItem title="Type">
-          {capitalize(database.type)}
+          {formatDatabaseType(database.type, image.version)}
+          {/* {`${capitalize(database.type)} ${image.version || ""}`} */}
```
Comment (joeclayallday) [inline src/ui/layouts/database-detail-layout.tsx:86]: Should we delete this comment?

Comment (grrowe) [reply src/ui/layouts/database-detail-layout.tsx:86]: yes, my bad

File: src/database/index.ts (hunk 2 above)
Comment (grrowe) [reply src/database/index.ts]: You want to move the capitalize function? I moved the `formatDatabaseType` into `src/deploy/database`

Comment (grrowe) [reply src/database/index.ts]: @neurosnap

Comment (neurosnap) [reply src/database/index.ts]: you did it right! sorry i wasn't clear, was just commenting on the file not the line in particular

**Review by neurosnap — APPROVED** [review]

**Review by joeclayallday — APPROVED** [review]

**Pull request merged**